assets/js/components/product-tag-control/style.scss

```diff
@@ -0,0 +1,16 @@
+.woocommerce-product-tags__operator {
```

Comment:
All styles in this files are only shown in the editor, right? Should they be moved to `editor.scss`?

Reply:
No, this is a component not a block. Other components use `style.scss`. Only the blocks use `editor.scss`

Reply:
Which other components?

`style.scss` will be included in the store frontend, so my understanding is that it shouldn't contain any styles from the admin, right? Otherwise we are polluting store's styles with unnecessary CSS rules. :worried: